wemake_python_styleguide/logic/complexity/overuses.py

```diff
@@ -101,13 +101,9 @@ def is_unary_minus(node: ast.AST) -> bool:
 
     We use this predicate to allow values
     like ``-some_value`` to be overused.
-
-    Although negative constants like ``-1``
```

Comment:
`-1` is fine. But, things like `-5` and `-37` should be reported.

Reply:
But they are already reported by "Found magic number" – should they be really reported again here?

Reply:
Yes, complexity rules are different from "Found magic number"

Reply:
I am looking at this: https://github.com/wemake-services/wemake-python-styleguide/blob/2377314181bc795e83ad4618a95a188cea86225b/wemake_python_styleguide/constants.py#L395

Should all negatives of those be excluded or only -1?

Reply:
@sobolevn what do you think?

Reply:
Only `-1`